Formula/scala.rb

```diff
@@ -17,17 +17,17 @@ class Scala < Formula
   option "with-docs", "Also install library documentation"
   option "with-src", "Also install sources for IDE support"
 
-  depends_on :java => "1.8+"
+  depends_on java: "1.8+"
```

Comment:
Hold on—I got the exact opposite message when I ran `brew audit --strict foo` which is why I made the change in the first place! Any idea why that is?
```
$ brew audit --strict scala
scala:
  * C: 20: col 14: Use the new Ruby 1.9 hash syntax.
Error: 1 problem in 1 formula
```